content/sql/concepts/window-functions/window-functions.md

```diff
@@ -16,7 +16,7 @@ CatalogContent:
 
 A window function performs a calculation over the inputted column and then returns the calculated value for each row. Window functions can be identified by their use of the `OVER` clause.
 
-In the simplest syntactic example, a function can be preformed over a given column as shown below:
+In the simplest syntactic example, a function can be performed over a given column as shown below:
```

Comment:
@garanews please revert all the changes for Swift entries. There's currently a larger PR our for Swift entry edits here: https://github.com/Codecademy/docs/pull/929

Reply:
You're free to comment on the above PR if you see anything that may need tweaking